release.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -eq 0 ]
+then
+	echo "No version supplied. Run \`make release VERSION=X.X\`"
+	exit 1
+fi
+
+version=$1
+
+echo $version | grep -Eq "^v[0-9]+\.[0-9]+\.[0-9]+$" && echo "match" || echo "no match"
+
+if ! echo $version | grep -Eq "^v[0-9]+\.[0-9]+\.[0-9]+$"
```

Comment:
computed twice. both above and below. Recommend storing in variable

Reply:
Whoops, the upper one was just for testing & I forgot to remove it. Thanks!